src/adapters/config/web-socket-config.ts

```diff
@@ -59,15 +59,17 @@ export class WebSocketConfigImpl implements WebSocketConfig {
     private readonly serializer: Serializer,
   ) {}
 
-  getProtocols(protocols: readonly string[] = []): string[] {
+  getHeaders(): Record<string, string> {
     if (
       this.props.useInsecureAccessToken ||
       this.props.accessToken == undefined
     ) {
-      return [...protocols];
+      return {};
     }
 
-    return [this.props.accessToken, ...protocols];
+    return {
+      Authorization: this.props.accessToken,
```

Comment:
This is wrong. The `Authorization` header requires the type, which is `Bearer`. So:

```
Authorization: `Bearer ${this.props.accessToken}`,
```

Reply:
Oh thank you for having a look, I'm fixing it in #1259